Add project description to README (#1)
natalix/cook-book-app: README.md

# Cook-Book-App

> This simple Cook-Book application will be using spoonacular API. It will include the following functionalities:

Reviewer: ```suggestion
This simple Cook-Book application will be using the spoonacular API. It will include the following functionalities:
```

- searching for recipies
- viewing recipe details

> - save recipie to Favourite List

Reviewer: ```suggestion
- saving recipe to Favourite List
```

> - create editable shoping list based on items selected from recipe

Reviewer: ```suggestion
- creating an editable shopping list based on items selected from recipe
```

<br>
<br>

---

## Designs

> Main page will be developed based on below wireframes - Desktop/Tablet/Mobile respectively:

Reviewer: ```suggestion
The main page will be developed based on the wireframes below - Desktop/Tablet/Mobile respectively:
```

<img src="readme_source/desktop.png" alt="Desktop main page" style="width:45%;"/>
<img src="readme_source/tablet.png" alt="Tablet main page" style="width:30%;"/>
<img src="readme_source/mobile.png" alt="Mobile main page" style="width:15%;"/>
<br>
<br>

> Recipe page will be developed based on below wireframes - Desktop and Tablet/Mobile respectively:

Reviewer: ```suggestion
The recipe page will be developed based on the wireframes below - Desktop and Tablet/Mobile respectively:
```

<img src="readme_source/desktop_tablet_recipe.png" alt="Desktop and tablet recipe page" style="width:65%;"/>
<img src="readme_source/mobile_recipe.png" alt="Mobile recipe page" style="width:25%;"/>
<br>
<br>

---

> Application is created to learn technologies listed below:

Reviewer: ```suggestion
This application is created to learn the technologies listed below:
```
